### ISSUE TITLE
Auto Advance doesn't work for chromecast

### ISSUE THREAD

**Issue by fuglphoenix** [KoljaWindeler/ytube_music_player#163]
**DEBUG Output**
```
2021-12-16 20:45:06 ERROR (MainThread) [custom_components.ytube_music_player.media_player] [S] async_sync_player ... media_player.reol: off -> playing
2021-12-16 20:45:06 ERROR (MainThread) [custom_components.ytube_music_player.media_player] detected app _id, B45F4572
2021-12-16 20:45:06 ERROR (MainThread) [custom_components.ytube_music_player.media_player] [E] async_sync_player
2021-12-16 20:45:21 ERROR (MainThread) [custom_components.ytube_music_player.media_player] [S] async_sync_player ... media_player.Reol [E]
2021-12-16 20:47:57 ERROR (MainThread) [custom_components.ytube_music_player.media_player] [S] async_get_track
2021-12-16 20:47:57 ERROR (MainThread) [custom_components.ytube_music_player.media_player] - Playing track nr 2 / 50
2021-12-16 20:47:57 ERROR (MainThread) [custom_components.ytube_music_player.media_player] [S] async_get_url
2021-12-16 20:47:57 ERROR (MainThread) [custom_components.ytube_music_player.media_player] [S] async_check_api [E]
2021-12-16 20:47:57 ERROR (MainThread) [custom_components.ytube_music_player.media_player] - try to find URL on our own
2021-12-16 20:47:57 ERROR (MainThread) [custom_components.ytube_music_player.media_player] - self decoded URL via cipher
2021-12-16 20:47:57 ERROR (MainThread) [custom_components.ytube_music_player.media_player] [E] async_get_url
2021-12-16 20:47:57 ERROR (MainThread) [custom_components.ytube_music_player.media_player] - forwarding url to player media_player.Reol
2021-12-16 20:47:57 ERROR (MainThread) [custom_components.ytube_music_player.media_player] [S] async_update_extra_sensor [E]
2021-12-16 20:47:58 ERROR (MainThread) [custom_components.ytube_music_player.media_player] [S] async_update_extra_sensor [E]
2021-12-16 20:47:58 ERROR (MainThread) [custom_components.ytube_music_player.media_player] [E] async_get_track
2021-12-16 20:48:13 ERROR (MainThread) [custom_components.ytube_music_player.media_player] [S] async_sync_player ... media_player.Reol [E]
2021-12-16 20:56:44 ERROR (MainThread) [custom_components.ytube_music_player.media_player] [S] async_sync_player ... media_player.reol: playing -> off
2021-12-16 20:56:44 ERROR (MainThread) [custom_components.ytube_music_player.media_player] media player got turned off
2021-12-16 20:56:44 ERROR (MainThread) [custom_components.ytube_music_player.media_player] turn_off
2021-12-16 20:56:44 ERROR (MainThread) [custom_components.ytube_music_player.media_player] [S] async_turn_off_media_player [E]
2021-12-16 20:56:44 ERROR (MainThread) [custom_components.ytube_music_player.media_player] [E] async_sync_player
```

**Describe the bug**
when playing a playlist on a chromecast audio. it stops after the first track.
my guess is that the chromecast doesn't switch to idle state after the track is finished.

**Version**
20211213.01

**To Reproduce**
1. select chromecast as speaker
2. select playlist
3. play
4. wait
5. sadness... 

Thanks

**Comment by KoljaWindeler** [comment]
Hmm that's super strange .. what Chromecast is that exactly? 
Have you read the readme https://github.com/KoljaWindeler/ytube_music_player#auto-advance

Specifically
https://github.com/KoljaWindeler/ytube_music_player#mpd-fix

Kolja

**Comment by fuglphoenix** [comment]
It is a Chromecast Audio.
I've been using it with ytube music for a year and had no problem until a couple of days ago.
I've monitored the cc in homeassistant and after the track is finished the state keeps saying "playing"
And yes I have looked at the documents, but it didn't help😔

**Comment by KoljaWindeler** [comment]
Ah, my bad 
I spotted in the log the transition from playing rato off but didn't check the timestamps. 9 min .. so that wasn't the end of track I guess ... Hmmm so if the state in Homeassistent doesn't change .. then we have a dependency problem that I can't solve.

Did the Chromecast integration get an update?

**Comment by fuglphoenix** [comment]
yes, it does look like it did get updated around the same time.
https://github.com/home-assistant/core/commits/dev/homeassistant/components/cast
i'll try and do a downgrade and see if that helps

**Comment by fuglphoenix** [comment]
i wonder if this is the same issue
https://github.com/home-assistant/core/pull/61786

**Comment by KoljaWindeler** [comment]
Very possible

**Comment by fuglphoenix** [comment]
i've downgraded to version 2021.11.5 and it works again.
so the ha update definitely change the status functionality
i've also noticed that the song progressbar in the lovelace media_player object was missing (in ha version 2021.12.2) and it also works again in 2021.11.5

**Comment by KoljaWindeler** [comment]
damn .. it still worked for me yesterday but today ALL my google device are broken as well ... 
I can confirm the same behavior: 
- state is always 'idle', not changing when the playback starts (and its playing for sure)
- media_position: always 0 
sadly there is nothing I can do to fix this. I'll downgrade my primary system as well, while keeping an eye out on the ticket mentioned above :(

**Comment by KoljaWindeler** [comment]
fixed in homeassistant 2021.12.4
